Dockerfile

```diff
@@ -1,22 +1,22 @@
 #
 # Dockerfile for Storm Mesos framework
 #
-FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+FROM mesosphere/mesos:0.25.0-0.2.70.ubuntu1404
 MAINTAINER Timothy Chen <tnachen@gmail.com>
 
 # build packages
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update
-RUN apt-get install -yq openjdk-7-jdk
+RUN apt-get install -yq openjdk-7-jdk maven wget
 
 # export environment
 ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
-# copy local checkout into /opt/storm
-ADD . /opt/storm
+ADD . /work
 
+WORKDIR /work
+RUN ./bin/build-release.sh main
```

Comment:
I'm not a docker expert, but if I'm reading this right then, the Docker image will *build* the release every time it is run.  Why is this being done?

I would really appreciate explicit enumeration of *all* behavior changes and their raison d'etre in this extensive PR.

e.g., why was ENTRYPOINT removed below?